### ISSUE TITLE
decidePolicyForNavigationResponse doesn't check that response is NSHTTPURLResponse

### ISSUE THREAD

**Issue by jordoh** [CRAlpha/react-native-wkwebview#199]
When loading an HTML page from a bundled file (e.g. `source={ { file: ... } }`), the `navigationResponse.response` is _not_ a `NSHTTPURLResponse`, which results in a SIGABRT when accessing the non-existent `allHeaderFields` property. This manifests (e.g. via crash reporting) as the following exception:
```
Application threw exception NSInternalInconsistencyException: Completion handler passed to -[RCTWKWebView webView:decidePolicyForNavigationResponse:decisionHandler:] was not called
```
![image](https://user-images.githubusercontent.com/450345/46103385-8c181600-c185-11e8-92f9-06799c2bba28.png)

I've tried passing `onNavigationResponse={ null }` to work around this, but `_onNavigationResponse` is still truthy even if I do, so the block in `decidePolicyForNavigationResponse` is still entered.

I'm happy to submit a PR to handle non-HTTP responses, given some guidance on what the desired handling would be. Should `_onNavigationResponse` be skipped in those cases, or called with different parameters? CC @DimitryDushkin

**Comment by insraq** [comment]
Should have been fixed with PR #195

**Comment by jordoh** [comment]
Great, thanks!
